docs/how-to/managing-content/ingest-guide.md

```diff
@@ -191,3 +191,92 @@ Our ingest form is organized into Paragraphs - organized groups of metadata fiel
     * The “add media” toggle, if turned on, means that after you save this Repository Item, you will be sent to the Media page. That is where you will add a file that will become part of the object.
     * If you choose not to add media, you will create a metadata-only record, but you can choose to add media to the item later via the object’s Media tab. Adding media later is a bit more complex than doing it from ingest, so it is recommended to start with a file in mind.
     * After Saving the item, you will be taken to the Media interface. Follow the [detailed instructions on adding media](/arca-docs/how-to/managing-content/media/).
+
+## Ingesting Newspaper Objects
+
+!!! warning
+    This section goes over how to create newspaper objects and ingest their individual issues manually. For **batch/bulk** ingest of newspaper objects, please review the documentation in the [Batch Operations](/arca-docs/how-to/batch-operations/getting-started.md) section of this site.
+
+Like the legacy platform, newspaper objects in the new Islandora use a unique View to display its members: 
+
+![<# Accordion foldout newspaper view in browser #>](/arca-docs/assets/newspaper-view.png "Screenshot")
+
+The accordion-style foldout displays issued according to the date listed in the `Date Issued/Date Created` field of the Origin Information section of the metadata form. While the Content Types are technically "Newspaper" and "Publication Issue", this format can be used for any periodical/serial-type objects.
+
+Newspaper parent objects and their child issues operate similarly to Collections and their children. The main difference is the value in the `Content Type` field. Following is a walkthrough for creating newspaper objects manually.
+
+### Newspaper Parent
+
+Start by creating your parent Newspaper object. 
+
+1. Go to **Content > Add Content > Repository Item**. 
+2. Fill in the metadata form as you normally would. 
+3. For `Content type`, select **Newspaper**. 
+4. Make sure to include a value under `Resource Type`, likely **Newspaper** or **Periodical**.
+5. Click Save.
+
+### Publication Issue (TIFF/JP2s)
```

Comment:
"Publication Issue: TIFF/JP2 Pages" (clarifying that this is for individual page images)